handlers/david.js

```diff
@@ -0,0 +1,116 @@
+const _      = require('lodash');
+const format = require('format');
+const helpers = require('../util/helpers.js');
+
+module.exports = {
+  name: 'david',
+  trigger: '.david',
+  handler: david,
+};
+
+// refers to the odds of modifying an individual character
+// NOT the odds of that particular modification happening to a word
+const config = {
+  addCharChance: .3,
+  capitalizationSwapChance: .4,
+  scrambleChance: .4,
+  removeCharChance: .2,
+}
+
+async function david(message) {
+  const messages = await helpers.getMessages(message);
+  const randomMessage = helpers.randomUserMessage(messages).content;
+  let words = randomMessage.split(" ");
```

Comment:
this appears to remove punctuation by default which I don't necessarily want to do. I'd have to use a regex to get the same behavior, and I don't think that's any better than this since this is so simple

Reply:
ah, fair call!